Gruntfile.js

```diff
@@ -1,73 +1,78 @@
-'use strict';
```

Comment:
When trying to run Grunt I get the following error:

Loading "Gruntfile.js" tasks...ERROR
>> SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode 

Re-adding 'use strict;' makes this error go away.